modules/process/MetaParse/MetaDataParser.nf

```diff
@@ -8,7 +8,7 @@ process MetaDataParser {
 
   output:
     path("*.sample_data.txt"), emit: MetaDataOutput
-    tuple val("placeHolder"), val(idTumor), val(idNormal), path("*.sample_data.txt"), emit: MetaData4Aggregate
+    tuple val(placeHolder), val(idTumor), val(idNormal), path("*.sample_data.txt"), emit: MetaData4Aggregate
```

Comment:
this doesn't look the same as other ones?

Reply:
because the value `val(placeHolder)` (not to be confused with `val("placeHolder")`) is also in the `input` context of the process, removing the quotes was just the easiest solution for allowing storeDir to work. i also did this for `RunNeoantigen` and `RunLOHHLA`. not all processes have `val(placeHolder)` in the input context, so that's why their fixes are different.

Reply:
Got it. Just want to make sure